adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -18,9 +18,9 @@
 (function(root, factory) {
```

Comment:
@edewit Could you remove the unrelated whitespace changes from your commit?

Reply:
you want to have spaces on the end of lines?

Reply:
No, just that the commit only includes the lines that are actually changed.

Reply:
just add `?w=1` on the url and you'll see only the important changes